pytest_inmanta/plugin.py

```diff
@@ -1137,6 +1137,14 @@ def enumerate_model(model: str):
         for key, blob in exporter._file_store.items():
             self.add_blob(key, blob)
 
+        for key, value in inmanta.resources.resource._resources.items():
```

Comment:
The only problem I see here is that we are now accessing the private property `_resources` from the resource class. In other words, this private property has now become part of the stable API. We should at least add a comment in core to make that clear.

Reply:
I've added it: 
https://github.com/inmanta/inmanta-core/pull/5841/files

Reply:
Nice!